packages/botonic-plugin-contentful/src/cms/contents.ts

```diff
@@ -428,6 +428,55 @@ export class ScheduleContent extends TopContent {
   }
 }
 
+export type OnFinish = Callback
+
+export class HandoffAgentEmail {
+  readonly type = 'AGENT_EMAIL'
+  constructor(readonly agentEmail: string) {}
+}
+
+export class HandoffAgentId {
+  readonly type = 'AGENT_ID'
+  constructor(readonly agentId: string) {}
+}
+
+export class HandoffQueue {
+  readonly type = 'QUEUE'
+  /**
+   * @param queue the queue id or name
+   */
+  constructor(readonly queue: Queue) {}
+}
+
+export type HandoffDestination =
+  | HandoffAgentEmail
+  | HandoffAgentId
+  | HandoffQueue
+
+/**
+ * Most CommonFields make no sense for Handoff.
+ * However, we decided to make it a TopContent since it does not depend on other content.
+ * Also CommonFields might be potentially useful.
+ */
+export class Handoff extends TopContent {
+  constructor(
+    readonly common: CommonFields,
+    readonly text: string,
```

Comment:
After taking a look at the test, I think that this `text` is the one to be displayed when doing the handoff, am I right?

Maybe we should consider adding here a new attribute to store the text to be displayed if the handoff can't be done (if a queue is closed or if an agent is not available for example), what do you think?